timeless/views.py

```diff
@@ -57,7 +56,8 @@ class CrudAPIView(views.MethodView):
 
     def get(self):
         """Calls the GET method."""
-        pass
+        model_id = request.form.get(self.url_lookup)
```

Comment:
@salgadobreno it does not work that way. id should be in args for `.get(self, id)`

Reply:
@vryazanov Noted, thanks!

Reply:
@salgadobreno 
```python
    def get(self, *args, **kwargs):
        """Calls the GET method."""
        model_id = kwargs.get(self.url_lookup)
```